components/layout_2020/display_list/mod.rs

```diff
@@ -175,6 +179,14 @@ pub(crate) struct DisplayListBuilder<'a> {
 }
 
 impl DisplayList {
+    #[cfg_attr(
+        feature = "tracing",
+        tracing::instrument(
+            name = "display_list::build",
+            skip_all,
+            fields(servo_profiling = true)
+        )
+    )]
```

Comment:
is there a way we can cut off the other trace events nested within this event unless we want detailed profiling of display list building?

Reply:
I think for now we are fine not having the detailed profiling of display_list, I have updated the code, now we are only capturing builder span and not the who nested trace events. If needed we can always expend to get the detailed profiling of the display_list

![image](https://github.com/user-attachments/assets/262cf0ae-bdff-4b9b-947d-f0aaa5b7dc15)

Reply:
that looks better, though there are still events under `display_list::build`, defined in StackingContextContent and StackingContext and glyphs(). are those events useful enough to keep?

![image](https://github.com/user-attachments/assets/ceeb49b3-ac1a-41bf-9745-fa88c09fb706)

Reply:
So, I went ahead and removed the once you suggested and some more of the events that doesn’t benefit us immediately. Initially, when I added it, I thought it could be useful, but of course, the benefit isn’t immediate. We can add it later if needed

<img width="813" alt="image" src="https://github.com/user-attachments/assets/5a30c5d5-59f2-4b0a-b1bd-28cc18825136">